generic_chooser/widgets.py

```diff
@@ -143,12 +120,9 @@ def render_input_html(self, name, value, attrs):
         return super().render_html(name, value, attrs)
 
     def render_html(self, name, value, attrs):
-        if WAGTAIL_VERSION >= (2, 12):
-            # From Wagtail 2.12, get_value_data is called as a preprocessing step in
-            # WidgetWithScript before invoking render_html
-            value_data = value
-        else:
-            value_data = self.get_value_data(value)
+        # From Wagtail 2.12, get_value_data is called as a preprocessing step in
+        # WidgetWithScript before invoking render_html
```

Comment:
The comment here could be removed as 2.12 has long been gone and it may just cause confusion if we're saying it support 4.1 + anyway.

Reply:
Removed comment
https://github.com/wagtail/wagtail-generic-chooser/pull/69/files#diff-acbfb7ec609db916584fe90e444eaff969a468b25ce8f06a77e6a6aa9d13732cL147-L149